Fix double conversion issues (#5080)
rotki/rotki: docs/changelog.rst

Changelog
=========

> * :feature:`5080` Temporarily fix the double conversion issue for manual latest price.

Reviewer: Please write user facing changelog entries.

1. Temporarly is wrong. The users don't care.
2. What is the double conversion issue? I am a user. I have no idea what you mean.

```suggestion
* :feature:`5080` In the manual balances section, the should no longer be any double conversion. So 1 euro should always be one euro.
```

> * :feature:`5080` In the manual balances section, the should no longer be any double conversion. So 1 euro should always be one euro.

Reviewer: ```suggestion
* :feature:`5080` For custom assets with custom price there should no longer be any double conversion. So 1 euro should always be one euro.
```
